src/map/status.c

```diff
@@ -12319,6 +12326,7 @@ void status_defaults(void) {
 	status->calc_mob_ = status_calc_mob_;
 	status->calc_pet_ = status_calc_pet_;
 	status->calc_pc_ = status_calc_pc_;
+	status->calc_pc_additional = status_calc_pc_additional;
```

Comment:
good point, didn't think of hooks, in that case the pointless function should remain, and in status_calc_pc you'd have this:
```
if( status->calc_pc_additional != status_calc_pc_additional ) status->calc_pc_additional(sd,opt);
```
however I'm unsure about the benefits of hooking to a pointless function.